b2sdk/file_lock.py

```diff
@@ -182,36 +182,57 @@ def __repr__(self):
             self.__class__.__name__, repr(self.mode.value), repr(self.retain_until)
         )
 
-class LegalHoldSerializer:
+
+@enum.unique
+class LegalHold(enum.Enum):
+    """Information about legalHold switch in a file."""
+
+    ON = 'on'
+    OFF = 'off'
+    UNSET = None  # this is the server default, as for now it is functionally equivalent to OFF
+    UNKNOWN = 'unknown'  # This one is used if the client is not authorized to read retention settings
+
+    def __bool__(self):
+        if self is LegalHold.UNKNOWN:
+            raise ValueError("Cannot determine the boolean value of an unknown Legal Hold")
+        return self is LegalHold.ON  # UNSET == FALSE for boolean determination
```

Comment:
whenever `is` is used in this PR, I try to imagine a future apiver which will need to stop inheritance due to MRO etc. Maybe it won't happen, but can't we just use `==` instead?